dbt_platform_helper/domain/versioning.py

```diff
@@ -68,34 +68,72 @@ def __init__(
         self.environment_variable_provider = environment_variable_provider
         self.platform_helper_version_override = platform_helper_version_override
 
-    def is_managed(self):
+    def is_auto(self):
         platform_config = self.config_provider.load_unvalidated_config_file()
         default_version = platform_config.get("default_versions", {}).get("platform-helper")
         return default_version == "auto"
 
-    def get_required_version(self):
+    def get_project_version(self):
+        version_status = self.get_version_status()
+        if self.is_auto():
+            return version_status.latest
+        else:
+            return self.get_default_version()
+
+    def get_default_version(self):
         platform_config = self.config_provider.load_unvalidated_config_file()
         required_version = platform_config.get("default_versions", {}).get("platform-helper")
-        self.io.info(required_version)
         return required_version
 
-    # Used in the generate command
+    def _check_auto_environment(self):
+        platform_helper_version_is_set_in_environment = self.environment_variable_provider.get(
+            PLATFORM_HELPER_VERSION_OVERRIDE_KEY
+        ) or self.environment_variable_provider.get("PLATFORM_HELPER_VERSION")
+        modules_override_is_set_in_environment = (
+            self.environment_variable_provider.get(
+                TERRAFORM_EXTENSIONS_MODULE_SOURCE_OVERRIDE_ENV_VAR
```

Comment:
Where are we setting the value of `TERRAFORM_EXTENSIONS_MODULE_SOURCE_OVERRIDE` in the platform-upgrade pipeline, I don't see it?

I see we set `TERRAFORM_ENVIRONMENT_PIPELINES_MODULE_SOURCE_OVERRIDE` & `TERRAFORM_CODEBASE_PIPELINES_MODULE_SOURCE_OVERRIDE` in their accompanying buildspecs in platform-orchestration

Reply:
TY!